src/ui/components/inputs/types.ts

```diff
@@ -3,10 +3,17 @@ import { TextInput as RNTextInput } from 'react-native'
 
 export type CustomTextInputProps = {
   isError?: boolean
+  inputHeight?: 'small' | 'tall'
+  LeftIcon?: React.FC
+  RightIcon?: React.FC
 }
```

Comment:
Je suis en train de me demander si ca vaut le coup de "dupliquer" ce type pour représenter les props des différents types d'inputs : password, text, search : 
```suggestion
export type SearchInputProps = {
  isError?: boolean
  inputHeight?: 'small' | 'tall'
  LeftIcon?: React.FC
  RightIcon?: React.FC
}
export type TextInputProps = {
  isError?: boolean
}
export type PasswordInputProps = {
  isError?: boolean
}
```

Reply:
je m'étais posé la même question. J'ai dupliqué search vs textinput dans le dernier commit. ça rend pas mal, à voir ce que t'en penses.

Reply:
J'ai commenté : https://github.com/pass-culture/pass-culture-app-native/pull/334#discussion_r555755384
Je suis ok avec ce que tu as fais, mais je n'aurais pas touché les props native par contre